src/components/elements/table/GenericTable.tsx

```diff
@@ -107,6 +107,10 @@ const GenericTable = <T extends { id: string }>({
   filterToolbar,
   noData = 'No data',
 }: Props<T>) => {
+  const columns = useMemo(
+    () => filter(columnProp || [], (c) => !c.hide),
```

Comment:
Should we prefer native js over lodash?
```suggestion
    () => (columnProp || []).filter((c) => !c.hide),
```

Reply:
Yeah we should, good call.. updated here and elsewhere